dashboard/src2/pages/NewServer.vue

```diff
@@ -84,6 +84,61 @@
 						</div>
 					</div>
 				</div>
+				<div
+					v-if="serverRegion && options.app_premium_plans.length > 0"
+					class="flex flex-col"
+				>
+					<div class="flex items-center justify-between">
+						<h2 class="text-sm font-medium leading-6 text-gray-900">
+							Plan Type
+						</h2>
+						<div>
+							<Button
+								link="https://frappecloud.com/pricing#dedicated"
+								variant="ghost"
+							>
+								<template #prefix>
+									<i-lucide-help-circle class="h-4 w-4 text-gray-700" />
+								</template>
+								Help
+							</Button>
+						</div>
+					</div>
+					<div class="mt-2 w-full space-y-2">
+						<div class="grid grid-cols-2 gap-3">
+							<button
+								v-for="c in [
+									{
+										name: 'Standard',
+										description:
+											'Suggested for small and medium sized companies'
```

Comment:
@NehaSan16 do you want to change the tooltip here and below to anything specific. This is shown on hover for plan type cards as shown in the screenshot